src/routes/(authentication)/login/+layout.svelte

```diff
@@ -1,11 +1,14 @@
 <script lang="ts">
+  import { initializeStores } from '@skeletonlabs/skeleton';
   import '../../../app.postcss';
   import { user } from '$lib/stores/User';
   import { goto } from '$app/navigation';
   import { onMount } from 'svelte';
   import Footer from '$lib/components/Footer.svelte';
   import Dots from '$lib/components/Dots.svelte';
 
+  initializeStores();
```

Comment:
Can we move this to the top level layout (`src/routes/+layout.svelte`) instead? I would rather it be only called a single time. And remove it from the `src/routes/(picsure)/+layout.svelte` file as well